lib/hammer_cli_foreman/architecture.rb

```diff
@@ -11,7 +11,12 @@ class ListCommand < HammerCLIForeman::ListCommand
         field :name, _("Name")
       end
 
-      build_options
+      extend_with(HammerCLIForeman::CommandExtensions::NoTaxonomies.new)
```

Comment:
Since this extension doesn't remove loc/org options expilictly, I'd change the name to `WithoutDefaults` or `NoDefaults`.